$NetBSD: patch-ac,v 1.4 2014/12/08 14:04:57 mef Exp $

limits.h is ISO C, unconditionally include it. Keep conditional for
sys/syslimits.h though.


--- chkdirs.c.orig	2006-01-19 23:40:21.000000000 +0000
+++ chkdirs.c
@@ -25,9 +25,8 @@
 
 */
 
-#if defined(__OpenBSD__) || defined(__FreeBSD__) || defined(__NetBSD__) || defined(__sun) || defined (hpux) || defined (__bsdi__) || defined (bsdi) || defined (__APPLE__)
 #include <limits.h>
-#elif defined(__APPLE__) && defined(__MACH__)
+#if defined(__APPLE__) && defined(__MACH__)
 #include <sys/syslimits.h>
 #endif
 
